src/core/dataset/data_types.ts

```diff
@@ -34,7 +34,12 @@ export let dataTypes: { [name in DataType]: DataTypeDescription } = {
     }
   },
   number: {
-    test: (x: string) => !isNaN(+x.replace(/\,/g, "")),
+    test: (x: string) => {
+      if (x === "null") {
```

Comment:
Can `x` be null/undefined here?

Reply:
No, the values come from string (it's part of string)so, only empty string suitable